nautobot_ssot/tests/device42/unit/test_models_nautobot_ipam.py

```diff
@@ -0,0 +1,511 @@
+"""Test DiffSync IPAM models for Nautobot."""
+
+from unittest.mock import MagicMock, patch
+from django.contrib.contenttypes.models import ContentType
+from django.forms import ValidationError
+from diffsync import DiffSync
+from nautobot.core.testing import TransactionTestCase
+from nautobot.dcim.models import Device, DeviceType, Interface, Location, LocationType, Manufacturer, Platform
+from nautobot.extras.models import Role, Status
+from nautobot.ipam.models import IPAddress, IPAddressToInterface, Namespace, Prefix, VLAN, VRF
+from nautobot_ssot.integrations.device42.diffsync.models.nautobot import ipam
+
+
+class TestNautobotVRFGroup(TransactionTestCase):
+    """Test the NautobotVRFGroup class."""
+
+    def setUp(self):
+        self.diffsync = DiffSync()
```

Comment:
What do you think about using the adapter directly here instead of the bare `DiffSync`?

Reply:
Using the direct adapter requires a lot more mocking and setup that isn't used. As I'm just testing the CRUD functions of the models directly I didn't think using the Device42Adapter would be necessary.